py3-pybind11-toolfile.spec

```diff
@@ -0,0 +1,22 @@
+### RPM external py3-pybind11-toolfile 1.0
+Requires: py3-pybind11
+
+%prep
+
+%build
+
+%install
+
+mkdir -p %{i}/etc/scram.d
+cat << \EOF_TOOLFILE >%{i}/etc/scram.d/py2-pybind11.xml
```

Comment:
@davidlange6 , please also update the xml file name. It should be py3-pybind11.xml